package.json

```diff
@@ -29,7 +29,7 @@
     "vite": "^4.3.9"
   },
   "peerDependencies": {
-    "o1js": "^1.8.0"
+    "o1js": "https://pkg.pr.new/o1-labs/o1js@0f8ff81"
```

Comment:
this is an automatic CI pre-release which is created for every push to main in o1js, using that for now (I also set up the same on `zksecurity/o1js` so that we could even use versions which aren't even landed on the main o1js repo yet)